runners/overlay-runner/config/config.go

```diff
@@ -1,68 +1,22 @@
 package config
 
 import (
-	"os"
-	"path"
-
-	"github.com/tinyci/ci-agents/clients/asset"
-	"github.com/tinyci/ci-agents/clients/log"
-	"github.com/tinyci/ci-agents/clients/queue"
-	"github.com/tinyci/ci-agents/config"
-	"github.com/tinyci/ci-runners/git"
+	fwConfig "github.com/tinyci/ci-runners/fw/config"
```

Comment:
this no longer conflicts with `ci-agents/config`. Should this still have an overridden import name?

Reply:
Hm. I'll peek deeper in a minute but I'm happy to change it too; I probably had both in there at some point.